server/database/models/index.js

```diff
@@ -13,10 +13,13 @@ db.stores = require('@database/models/stores').model(sequelize, Sequelize.DataTy
 db.addresses = require('@database/models/addresses').model(sequelize, Sequelize.DataTypes);
 db.suppliers = require('@database/models/suppliers').model(sequelize, Sequelize.DataTypes);
 db.users = require('@database/models/users').model(sequelize, Sequelize.DataTypes);
+db.users = require('@database/models/students').model(sequelize, Sequelize.DataTypes);
+db.users = require('@database/models/subjects').model(sequelize, Sequelize.DataTypes);
 
 db.purchasedProducts = require('@database/models/purchased_products').model(sequelize, Sequelize.DataTypes);
 db.storeProducts = require('@database/models/store_products').model(sequelize, Sequelize.DataTypes);
 db.supplierProducts = require('@database/models/supplier_products').model(sequelize, Sequelize.DataTypes);
+db.supplierProducts = require('@database/models/students_subjects').model(sequelize, Sequelize.DataTypes);
```

Comment:
Why is this supplierProducts ? Should be studentSubjects right.

Reply:
Updated accordingly.